README.md

```diff
@@ -63,3 +63,27 @@ module.exports = webpackConfig;
 **Compiled with errors**
 
 ![image](https://user-images.githubusercontent.com/1018196/35707809-14e12926-0760-11e8-9f67-af1eff6048ae.png)
+
+## Customizing
+
+See `index.js` for examples.
+
+- `window.__webpackEventColors__` [Object] Override default colors.
+- `window.__webpackEventStyles__` [Object] Override default CSS.
+- `window.__webpackEventElem__` [DOMElement] DOM Element to apply the style on. Default: `body`.
+- `{webpackEventColor}` Use this placeholder in your CSS rules.
```

Comment:
I was a bit confused about the webpackEventColor

This would deserve an example e.g 

## Example

file: ./src/custom-webpack-status-bar.js
```js
window.__webpackEventColors__ = {
   errors: '#ff0000'
}

window.__webpackEventStyles__ = {
   position: 'fixed',
   top: '0px',
   color: '{webpackEventColor}',
   height: '3px',
   border: 'none',
}

window.__webpackStatusElemSelector = `#webpackStatusBar`;

require(`webpack-dev-server-status-bar`);
```

This will render something like this

<screenshot example>